setting-up-multifactor-authentication.md

```diff
@@ -60,3 +60,25 @@ Steps to change your MFA level:
 in the _multifactor authentication_ section, you will see a dropdown menu.
     ![Multifactor section at profile edit page](/images/changing_mfa_step1.png){:class="t-img"}
 2. Select your intended option, enter OTP access code from your authenticator app and click **Update**
+
+## Using recovery codes and re-setup a previously enabled MFA
+
+You might be in a situation where you no longer have access to your rubygems account in an
+authenticator app. This can happen if you loose or upgrade your phone or accidentally deleted
```

Comment:
Typo fix (loose -> lose) + change the first `or` to a `/` so it's a bit clearer how the grouping is intended.

```suggestion
authenticator app. This can happen if you lose/upgrade your phone or accidentally deleted
```